components/onboarding-modal/OnboardingModal.js

```diff
@@ -0,0 +1,224 @@
+    }
+  }
+
+  setStep = queryStep => {
+    if (queryStep === undefined) {
+      this.setState({ step: 0 });
+    } else if (queryStep === 'administrators') {
+      this.setState({ step: 1 });
+    } else if (queryStep === 'contact') {
+      this.setState({ step: 2 });
+    }
+  };
+
+  addAdmins = members => {
+    this.setState({ members });
+  };
+
+  addContact = (name, value) => {
+    this.setState(state => ({
+      collective: {
+        ...state.collective,
+        [name]: value,
+      },
+    }));
+  };
+
+  submitAdmins = async () => {
+    try {
+      this.setState({ isSubmitting: true, error: null });
+      await this.props.EditCollectiveMembers({
+        collectiveId: this.props.collective.id,
+        members: this.state.members.map(member => ({
+          id: member.id,
+          role: member.role,
+          member: {
+            id: member.member.id,
+            name: member.member.name,
+          },
+        })),
+      });
+    } catch (e) {
+      this.setState({ isSubmitting: false, error: getErrorFromGraphqlException(e) });
+    }
+  };
+
+  submitContact = async () => {
+    const collective = {
+      ...this.state.collective,
+      id: this.props.collective.id,
+    };
+    try {
+      this.setState({ isSubmitting: true, error: null });
+      await this.props.EditCollectiveContact({
+        collective,
+      });
+    } catch (e) {
+      this.setState({ isSubmitting: false, error: getErrorFromGraphqlException(e) });
+    }
+  };
+
+  submitCollectiveInfo = async () => {
+    await this.submitContact();
+    await this.submitAdmins();
+    Router.pushRoute('editCollective', { slug: this.props.collective.slug, section: 'info' });
+  };
+
+  render() {
+    const { collective, LoggedInUser } = this.props;
+    const { step, isSubmitting } = this.state;
+
+    return (
+      <Flex flexDirection="column" alignItems="center" py={[4]}>
+        <StepsProgressBox mb={[3, null, 4]} width={0.8}>
+          <OnboardingStepsProgress step={step} handleStep={step => this.setState({ step })} slug={collective.slug} />
+        </StepsProgressBox>
+        <Image src="/static/images/create-collective/communityIllustration.png" alt="Welcome!" />
+        <OnboardingContentBox
+          step={step}
+          collective={collective}
+          LoggedInUser={LoggedInUser}
+          addAdmins={this.addAdmins}
+          addContact={this.addContact}
+        />
+        <StyledHr my={4} borderColor="black.300" width="100%" />
+        <OnboardingNavButtons
+          step={step}
+          slug={collective.slug}
+          submitCollectiveInfo={this.submitCollectiveInfo}
+          loading={isSubmitting}
+        />
+      </Flex>
+    );
+  }
+}
+
+// GraphQL for editing Collective admins info
+const MemberFieldsFragment = gql`
+  fragment MemberFieldsFragment on Member {
+    id
+    role
+    member {
+      id
+      name
+    }
+  }
+`;
+
+const editCoreContributorsQuery = gql`
```

Comment:
should usually be named `Mutation` and not `Query` no?

Reply:
That does make sense, but just following the pattern in e.g. NewCreateCollective where there's a `query` that is fed into the `mutation` that does the actual `props: ({ mutate }) => ...` stuff. 

```
const createCollectiveQuery = gqlV2`
  mutation createCollective(
    $collective: CollectiveCreateInput!
    $host: AccountReferenceInput
    $automateApprovalWithGithub: Boolean
  ) {
    createCollective(collective: $collective, host: $host, automateApprovalWithGithub: $automateApprovalWithGithub) {
      name
      slug
      tags
      description
      githubHandle
    }
  }
`;

const addCreateCollectiveMutation = graphql(createCollectiveQuery, {
  options: {
    context: API_V2_CONTEXT,
  },
  props: ({ mutate }) => ({
    createCollective: async ({ collective, host, automateApprovalWithGithub }) => {
      return await mutate({
        variables: { collective, host, automateApprovalWithGithub },
        awaitRefetchQueries: true,
        refetchQueries: [{ query: getLoggedInUserQuery }],
      });
    },
  }),
});
```

Should I change it in both places?

Reply:
Will address this in next PR

Reply:
Yes